lib/utils/azure_queues.ts

```diff
@@ -13,13 +16,28 @@ import {
   toRuntimeError,
   TransientError
 } from "io-functions-commons/dist/src/utils/errors";
+import { ReadableReporter } from "italia-ts-commons/lib/reporters";
 
-export interface IQueueMessage extends QueueService.QueueMessageResult {
-  readonly id: string;
-  readonly nextVisibleTime: string;
-  readonly popReceipt: string;
-  readonly queueTrigger: string;
-}
+export const QueueMessage = t.intersection([
+  t.interface({
+    id: t.string,
+    popReceipt: t.string
+  }),
+  t.partial({
+    dequeueCount: t.number,
```

Comment:
you may consider to put dequecount into the required params above and get rid of the following check:

https://github.com/teamdigitale/io-functions/blob/f160fcaed4bbd13f8a04e71fab5cea438ac570d6/lib/utils/azure_queues.ts#L128

Reply:
is that field always defined? (I didn't find any doc)